package.json

```diff
@@ -85,6 +85,7 @@
     "webpack-dev-server": "^3.1.4"
   },
   "dependencies": {
+    "babel-plugin-transform-runtime": "^6.23.0",
```

Comment:
I think I know why, but just curious, why this and the linter change?

Reply:
This is so we can transpile native ES6 promises and async functions to ES5. The window.createImageBitmap returns a promise, so I added these babel plugins such that I could write a polyfill for brush for IE/Edge/Safari.

Reply:
Shouldn't we use babel-runtime as the dependency, and transform-runtime as the dev dependency? https://babeljs.io/docs/en/babel-plugin-transform-runtime.html#installation

Reply:
Hmm, you're right.